x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/optimizer/OptimizerTests.java

```diff
@@ -1023,4 +1022,43 @@ public void testReplaceAttributesWithTarget() {
         gt = (GreaterThan) and.left();
         assertEquals(a, gt.left());
     }
+
+    //
+    // ReplaceSumWithStats rule
+    //
+    public void testSumIsReplacedWithStats() {
+        FieldAttribute fa = getFieldAttribute();
+        Sum sum = new Sum(EMPTY, fa);
+        
+        Alias sumAlias = new Alias(EMPTY, "sum", sum);
+        
+        Aggregate aggregate = new Aggregate(EMPTY, FROM(), emptyList(), asList(sumAlias));
+        LogicalPlan optimizedPlan = new Optimizer().optimize(aggregate);
+        assertEquals(Aggregate.class, optimizedPlan.getClass());
```

Comment:
```suggestion
        `assertTrue(optimizedPlan instanceof Aggregate);`
```

Reply:
We have a mix of those assertions, I wouldn't mind keeping it as is.